docs/main/user-guides/migration/legacy-storage-to-new-storage/device-to-lvm.md

```diff
@@ -0,0 +1,144 @@
+---
+id: device-to-lvm
+title: Migration from OpenEBS Local PV Device to OpenEBS Local PV LVM
+keywords:
+ - Migration from OpenEBS Local PV Device to OpenEBS LVM Local PV
+ - Local PV Device to Local PV LVM
+ - Local PV Device to Local PV ZFS
+ - Local PV Rawfile to Local PV LVM
+ - Local PV Rawfile to Local PV ZFS
+description: This section outlines the process of migrating OpenEBS Local PV Device to OpenEBS LVM Local PV.
+---
+
+:::info
+The following steps are an example about migrating from legacy storage to new storage.
+You can also migrate OpenEBS Local PV Device to OpenEBS Local PV ZFS & OpenEBS Local PV Rawfile to OpenEBS Local PV LVM or Local PV ZFS using the steps below.
```

Comment:
`OpenEBS Local PV Rawfile` removed this? We don't have rawfile anymore, only zfs and lvm.

Reply:
We rephrased the statement to say that "A user with OpenEBS Local PV Rawfile can migrate to OpenEBS Local PV LVM or Local PV ZFS."
 
We have separated the sentence into two parts as below to avoid confusion:
 "Info:
The following steps are an example about migrating from legacy storage to new storage.
You can also migrate OpenEBS Local PV Device to OpenEBS Local PV ZFS. Similarly, you can migrate OpenEBS Local PV Rawfile to OpenEBS Local PV LVM or Local PV ZFS using the steps below."